src/CppUTest/TestRegistry.cpp

```diff
@@ -50,6 +50,7 @@ void TestRegistry::runAllTests(TestResult& result)
     result.testsStarted();
     for (UtestShell *test = tests_; test != NULL; test = test->getNext()) {
```

Comment:
@basvodde it is in fact a new interface we are adding to UtestShell, and that's why in my opinion it needs to be there (like @cuitoldfish pointed out). It is the same as runInSeparateProcess_ or currentRepetition_. We need to be able to tell UtestShell from the "outside world" how it is supposed to behave.